packages/api/src/external/commonwell/patient.ts

```diff
@@ -325,14 +304,14 @@ export async function update(
           commonwellPatient,
           commonwellPatientId,
           patientRefLink,
-          storeIds: getStoreIdsFn(patient.id, patient.cxId),
+          storeIdsAndStatus: getStoreIdsAndStatusFn(patient.id, patient.cxId),
           getOrgIdExcludeList,
         });
         return;
       }
       //eslint-disable-next-line @typescript-eslint/no-explicit-any
     } catch (err: any) {
-      log(
+      debug(
```

Comment:
nit: error handling logs are not debug level, so I'd keep these as log instead of debug